src/App.jsx

```diff
@@ -1,40 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs';
 
-export const tabs = [
-  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-];
+export const App = () => {
+  const [selectedTabId, setSelectedTabId] = useState('1');
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
+  const tabSelected = (tabId) => {
+    setSelectedTabId(tabId);
+  };
 
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-          </li>
+  return (
+    <div className="section">
+      <h1 className="title">
+        {`Selected tab is Tab ${selectedTabId.replace('tab-', '')}`}
+      </h1>
 
-          <li data-cy="Tab">
-            <a href="#tab-2" data-cy="TabLink">Tab 2</a>
-          </li>
-
-          <li data-cy="Tab">
-            <a href="#tab-3" data-cy="TabLink">Tab 3</a>
-          </li>
-        </ul>
-      </div>
-
-      <div className="block" data-cy="TabContent">
-        Some text 1
-      </div>
+      <Tabs onTabSelect={tabSelected} />
```

Comment:
Also, you have to implement the next requirement:
![image](https://github.com/mate-academy/react_tabs-js/assets/25587299/da011184-7ee4-4cf6-905d-76d4f6119c57)

Reply:
And this one:
![image](https://github.com/mate-academy/react_tabs-js/assets/25587299/588e2f12-717b-47df-8027-1c1e8f7d0b2d)